react-components/src/components/RevealToolbar/Help/elements.ts

```diff
@@ -0,0 +1,106 @@
+/*!
+ * Copyright 2023 Cognite AS
+ */
+import { Body, Colors, Detail, Title } from '@cognite/cogs.js';
+import styled from 'styled-components';
+import { Mouse } from '../../Graphics/Mouse';
+
+export const SectionContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  gap: 8px;
+  width: fit-content;
+  max-width: fit-content;
+`;
+
+export const SectionTitle = styled(Title).attrs({ level: 3 })`
+  color: #ffffff;
+`;
+
+export const SectionSubTitle = styled(Title).attrs({ level: 5 })`
+  color: #ffffff;
+`;
+
+export const SectionContent = styled.div`
+  align-items: center;
+  display: flex;
+  flex-direction: row;
+  height: 100%;
+`;
+
+export const InstructionText = styled(Body).attrs({
+  level: 3,
+  strong: true
+})`
+  color: rgba(255, 255, 255, 0.9);
+  text-align: center;
+  font-size: 14px;
+  font-weight: 400;
+  line-height: 20px;
+`;
+
+export const InstructionDetail = styled(Detail)`
+  font-family: Inter;
+  font-size: 14px;
+  font-weight: 400;
+  line-height: 20px;
+  letter-spacing: -0.006em;
+  text-align: left;
+  color: ${Colors['text-icon--interactive--disabled--inverted']};
+  white-space: pre-line;
+`;
```

Comment:
These looks like Cogs.js typography styling, no? Can't we use Cogs.js typography here instead?